retis/src/collect/collector/nft/nft.rs

```diff
@@ -123,6 +123,10 @@ impl Collector for NftCollector {
         Ok(Self::default())
     }
 
+    fn known_kernel_types(&self) -> Option<Vec<&'static str>> {
```

Comment:
thanks @atenart for taking a look.
This definitely makes sense as per probe options left this uncovered.
I'm now only thinking about the mismatching behavior between --stack (which theoretically can produce stack traces everywhere) vs the per-probe option version. Per-probe option doesn't aim at mimicking that option, still so far we have this difference.